cli/tools/bundle/mod.rs

```diff
@@ -127,13 +128,33 @@ pub async fn bundle_init(
 
   let resolved_entrypoints =
     resolve_entrypoints(&resolver, &init_cwd, &bundle_flags.entrypoints)?;
-  let _ = plugin_handler
-    .prepare_module_load(&resolved_entrypoints)
-    .await;
 
-  let roots =
-    resolve_roots(resolved_entrypoints, sys, npm_resolver, node_resolver);
-  let _ = plugin_handler.prepare_module_load(&roots).await;
+  let input = if resolved_entrypoints.len() == 1
+    && resolved_entrypoints[0].as_str().ends_with(".html")
+  {
```

Comment:
What happens is someone does `deno bundle --outdir=dist/ main.ts index.html`. Will bundling `index.html` fail/error/panic?

Reply:
supported properly now